build_sysext

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+# Copyright (c) 2023 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can
+# be found in the LICENSE file.
+#
+# Script to generate sysext. See systemd-sysext(8). Prerequisite is
+# that you've run build_packages and build_image.
+
+SCRIPT_ROOT=$(dirname "$(readlink -f "$0")")
+. "${SCRIPT_ROOT}/common.sh" || exit 1
+
+# Script must run inside the chroot
+assert_inside_chroot
+assert_root_user
+
+# All these are used to set up the 'BUILD_DIR' variable
+DEFINE_string board "${DEFAULT_BOARD}" \
+  "The board to build a sysext for."
+DEFINE_string output_root "${DEFAULT_BUILD_ROOT}/images" \
+  "Directory in which to place sysext images and intermediate files"
+DEFINE_string group "developer" \
+  "The normal image build group"
+DEFINE_integer build_attempt 1 \
+  "The build attempt for this build."
+
```

Comment:
These flags are needed only because this script includes `build_image_util`, which doesn't make much sense here - we are not using it for anything else but the `BUILD_DIR` variable. I'd say that this is something that we could get directly from a flag like `--build_dir`.

```suggestion
DEFINE_string build_dir '' \
  "Directory used for building the sysext image, must exist, but should be empty."
```